lib/vdsm/tool/configurators/multipath.py

```diff
@@ -327,22 +327,20 @@ def configure():
     commands.run([_MULTIPATHD.cmd, "reconfigure"])
 
 
-def _check_conf_rev(mpathconf):
+def _check_conf_rev(first, second):
     """
     The multipath configuration file should contain a tag in form
     "RHEV REVISION X.Y" for this check to succeed.
     If the tag above is followed by tag "RHEV PRIVATE" the configuration
     should be preserved at all cost.
 
     Args:
-        mpathconf (List[str]): Lines in the multipath daemon
-        configuration file.
+        first (str): First line in the multipath daemon configuration file.
+        second (str): Second line in the multipath daemon configuration file.
```

Comment:
Passing here the first lines of the file is not an improvement. Please keep the code as is, passing the contents of the file as lines.

Reply:
OK. But it can be a list with two elements so that we can do:

     first, second = mpathconf

and not raise (with `readline()`).

Reply:
But mpathconf should be the entire file. The caller should not do the work for the function.